xiaoyifang/goldendict-ng
# fix: soundDir index not updating when sounds modified

Related https://github.com/xiaoyifang/goldendict-ng/issues/1417

Just compare modifications of index file and soundDir's.

This works at least on Linux+Btrfs. A folder's modification time will update if a file changes inside that folder.

To test: 

* add a sound folder and modify file names/contents/folder paths etc.
* click "edit -> dictionry -> Rescan now" -> sound dir will reindex

[sounds.zip](https://github.com/xiaoyifang/goldendict-ng/files/14662621/sounds.zip)

A little flaw:

GD doesn't have active folder monitoring. The "Rescan Now" only appears in “Files” but not in “Sound Folders”

![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/2c2791b5-6d20-45bd-b49c-6f101f749d1c)
